lib/tasks/scripts/telemedicine_reports.rb

```diff
@@ -84,6 +84,9 @@ def assemble_report_data
       "Patients with High Blood Sugar",
       "Patients with High BP or Sugar",
       "Teleconsult Button Clicks",
+      "Teleconsultation MVP Requests",
```

Comment:
Do we want to call it "MVP" in the report? Is there some other name that this report's consumers would prefer? I can't think of any off the top of my head, but we could check with Mahima.

Reply:
Briefed Mahima on this, she'll take a look and get back to me in the next few days.